2_observations/src/munge_surface_area_data.R

```diff
@@ -0,0 +1,65 @@
+read_monthly_surface_area_data <- function(res_sf_dir) {
+
+  # check if data exists
+  # if not, throw error indicating how to download
+  if (!file.exists("2_observations/in/009130")) {
+    stop("You must download reservoir monthly shapefile folder (0) data from:
+         \nhttp://umnlcc.cs.umn.edu/realsat
+         \nunzip the folder,
+         \nthen locate folder '009130' and unzip the folder into '2_observations/in'.")
+  }
+
+  if (!file.exists("2_observations/in/573567")) {
+    stop("You must download reservoir monthly shapefile folder (7) data from:
+         \nhttp://umnlcc.cs.umn.edu/realsat
+         \nunzip the folder,
+         \nthen locate folder '573567' and unzip the folder into '2_observations/in'.")
+  }
+
+  # point to the data directory
+  # getting list of shape files in this directory
+  # getting the directory for cannonsville reservoir shape file.
+  cannonsville_sf_dir <- list.files("2_observations/in/573567",
+                                    pattern = ".shp$", full.names = TRUE)
+  # getting the directory for pepacton reservoir shape file.
+  pepacton_sf_dir <- list.files("2_observations/in/009130",
+                                pattern = ".shp$", full.names = TRUE)
```

Comment:
There's another opportunity to make your code more DRY in this code chunk (lines 22-26)